beacon-chain/sync/subscriber_committee_index_beacon_attestation.go

```diff
@@ -0,0 +1,28 @@
+package sync
+
+import (
+	"context"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/prysmaticlabs/prysm/beacon-chain/core/helpers"
+)
+
+func (r *RegularSync) committeeIndexBeaconAttestationSubscriber(ctx context.Context, msg proto.Message) error {
+
+	return nil
+}
+
+func (r *RegularSync) currentCommitteeIndex() int {
+	state, err := r.chain.HeadState(context.Background())
+	if err != nil {
+		panic(err)
+	}
+	if state == nil {
+		return 0
```

Comment:
Why wouldn't this be panic?

Reply:
No, it was a panic and crashing on genesis chain start.